packages/scaffold-ui/src/partials/w3m-alertbar/styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    display: block;
+    position: absolute;
+    opacity: 1;
+    top: 11px;
+    left: 16px;
```

Comment:
Can we use spacing variables for horizontal values? Also why the top value doesn't follow our spacing system? Maybe we need to validate with the designer?

Reply:
Good catch, we should probably do the same for [snackbar](https://github.com/reown-com/appkit/blob/main/packages/scaffold-ui/src/partials/w3m-snackbar/styles.ts#L9).